pkg/providers/common/common.go

```diff
@@ -26,7 +26,18 @@ const (
 	publicKeyFileName  = "eks-a-id_rsa.pub"
 )
 
-func GetAuditPolicy() string {
+func GetAuditPolicy(kubeVersion v1alpha1.KubernetesVersion) string {
+	v1policy, err := v1alpha1.GreaterOrEqual(kubeVersion, v1alpha1.Kube124)
+	if err != nil {
+		return ""
+	}
+	if v1policy {
+		auditPolicyv1, err := GetAuditPolicyv1Yaml()
+		if err != nil {
+			return ""
```

Comment:
why are we ignoring the error? this method should probably return an error

Reply:
This method is called in `buildTemplateMapCP` which does not return any errors, just a map of values. should we log this as an error instead and return empty value for the audit policy?

Reply:
Addd I see. I would return and error from `buildTemplateMapCP`